File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int ans = 0;
        int r = s.size() - 1;
        while(r >= 0)
        {
            if(s[r] != ' ')
            ans++;
            else if(ans>0)
            return ans;
            r--;
        }
        return ans;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i , c = len(s)-1 , 0
        while s[i] == " ":
            i-=1
        while i>=0 and s[i]!=" ":
            c+=1
            i-=1
        return c
        